js/views/modals/About/Contributors.js

```diff
@@ -1,35 +1,302 @@
 import loadTemplate from '../../../utils/loadTemplate';
 import baseVw from '../../baseVw';
+import regeneratorRuntime from 'regenerator-runtime';
 
-const names = [
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
+let names;
+let fetchNames;
+
+// Linter notes:
+  // linter will not be happy with some things which also work:
+  // no-use-before-define -- because of const variable hoisting and invocation time
+  // this works and is a way to share data in closures. The linter is also unhappy.
+{
+  // proof of concept for Dynamic Contributors List
+  // see the comments for more information
+
+  const obGitHubName = 'OpenBazaar';
+  const base = 'https://api.github.com';
+
+  // this is from the GitHub entity created
+  // for purposes of testing this code: theob1ioaccount000
+  // it has no access to anything except public information
+  // because of the limit of 5K calls
+  // there can only be around 600 ( 25 * 24 ) such calls to the
+  // contributor_names process per day ( each call makes around 200 requests )
```

Comment:
This is neat, but we probably want to just maintain a static list. The list won't change very often, and it seems fragile to count on a special github account to access it. And we should hope we'll be successful enough to exceed the call limit.

(I see you have made comments along those lines later on, and this was just for testing)